### DISCUSSION TITLE
Rendering SLS 'base:grafana' failed: Jinja variable 'dict object' has no attribute 'pipeline_overview_nontc'

### DISCUSSION THREAD

**Discussion by njohnson-q** [Security-Onion-Solutions/securityonion#6547] [Unsupported Versions] [answered]
Version: 2.3.70
On prem
ISO install
1 manager 1 search
So-Status show all running.
Yes, cannot render grafana sls. Error below
No errors on Grid page

Provide applicable logs. For example: if you are having problems during soup, provide /root/soup.log

Issue: Tried upgrading to version 2.3.90 with the latest hotfix to resolve Log4J issues. When upgrading I get the error:

Running a highstate. This could take several minutes.
local:
    Data failed to compile:
----------
    Rendering SLS 'base:grafana' failed: Jinja variable 'dict object' has no attribute 'pipeline_overview_nontc'
Soup failed with error 1: Unhandled error
Unhandled error occured, please check /root/soup.log for details.

[soup.log](https://github.com/Security-Onion-Solutions/securityonion/files/7707739/soup.log)

**Comment by m0duspwnens** [comment, marked as answer]
@njohnson-q Can you check the contents of `/opt/so/saltstack/default/salt/grafana/defaults.yaml` and verify that `pipeline_overview_nontc` is defined? If it is, do you have a `defaults.yaml` file located in `/opt/so/saltstack/local/salt/grafana/` where it isn't defined?

**Reply by njohnson-q** [reply]
That fixed the issue thank you!
